packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping-anchor.md

```diff
@@ -0,0 +1,197 @@
+# Mapping Anchor Table Column
+
+## Overview
+
+The `nimble-table-column-mapping-anchor` is a component that supports rendering specific number, boolean, or string values as an icon or a spinner, with an accompanying text anchor element. The mappings are defined by child elements of `nimble-mapping-icon`, `nimble-mapping-spinner`, or `nimble-mapping-empty`.
+
+### Background
+
+[Icon with anchor column](https://github.com/ni/nimble/issues/2531)
+
+### Features
+
+- Supported input:
+    - string
+    - number
+    - boolean
```

Comment:
This statement is a bit confusing (at least to me). Since there are multiple inputs to the column that support different types, it might be more helpful to split out the "Supported input" section into what is supported for a given input. Something like:

- Supported input:
    - string, number, or boolean mapping
    - string label
    - string href